apps/omstillingsstoenad-ui/src/components/soknad/5-mer-om-situasjonen-din/fragmenter/SelvstendigInfokort.tsx

```diff
@@ -87,16 +96,17 @@ const SelvstendigInfokort = memo(({ lengde, index, fjern }: Props) => {
                 />
             </SkjemaElement>
 
-            {endretArbeidssituasjon === IValg.JA && (
+            <div style={{ display: visEndretArbeidssituasjon ? 'block' : 'none' }}>
```

Comment:
Bruk heller en av aksel sine primitiver [https://aksel.nav.no/komponenter/primitives/hide](https://aksel.nav.no/komponenter/primitives/hide)

Reply:
Ser den er markert som uegnet for _Gjemme elementer basert på state_